### PR TITLE
Updating README file

### PR DESCRIPTION
**Pull request by lisa-liao**
I will incrementally update rally-staging with Rally code so its not 1500 lines at once. At the end of that process rally-candidate will have all reviewed code ready to merge into flutter/samples.

### CODE REVIEW

**Review by clocksmith — APPROVED** [review]

File: rally/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+**This sample is not currently in a finished state. We're in the process
+of building it out.**
+
+## Goals
+
+* Show how to customize a tab bar.
+* Show how to create reusable custom widgets with composition and custom painting.
+* Show how to create an app with tabs and child navigation screens.
+
+## Widgets / APIs
```
Comment (redbrogdon) [inline rally/README.md]: This is not a section we normally include with our samples. The "important bits" section you've got below is doing the work of explaining in broad terms what people should expect and focus on, so you don't need this list.

File: rally/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+### `/sections/*`
+
+These are the main sections for the tab views and the child screen with the details and line chart.
+The financial entity is a reusable screen that is the base for the accounts, bills, and budgets
+screens.
+
+#### Notice
+
+* Theming is passed as a parameter in the constructor of `MaterialApp` (`theme:`).
```
Comment (redbrogdon) [inline rally/README.md]: I don't know that this section is necessary here. We do all of these things in other sample apps, and you can always draw a little extra attention to them with inline comments in the code.

**Review by redbrogdon — CHANGES_REQUESTED** [review]

**Comment by redbrogdon** [comment]
Since you're going to be adding more Material studies to the repo, we went ahead and made a subdirectory specifically for them:

https://github.com/flutter/samples/pull/128

You might want to rebase on the new code and move this project into that directory to avoid the merge headache later.

File: rally/README.md (hunk 1 above)
Comment (lisa-liao) [reply rally/README.md]: Done

File: rally/README.md (hunk 2 above)
Comment (lisa-liao) [reply rally/README.md]: Done

**Pull request merged**